updating/updating-restricted-network-cluster.adoc

```diff
@@ -38,11 +38,31 @@ include::modules/cli-installing-cli.adoc[leveloffset=+3]
 
 // this file doesn't exist, so I'm including the one that should pick up more changes from Clayton's PR - modules/installation-adding-mirror-registry-pull-secret.adoc[leveloffset=+1]
 
-include::modules/installation-adding-registry-pull-secret.adoc[leveloffset=+2]
+include::modules/installation-adding-registry-pull-secret.adoc[leveloffset=+3]
 
-include::modules/update-mirror-repository.adoc[leveloffset=+2]
+[id="update-mirror-repository_updating-restricted-network-cluster"]
+=== Mirroring the {product-title} image repository
 
-include::modules/machine-health-checks-pausing.adoc[leveloffset=+3]
+Before you update a cluster on infrastructure that you provision in a restricted network, you must mirror the required container images into that environment. You can also use this procedure in unrestricted networks to ensure your clusters only use container images that have satisfied your organizational controls on external content.
+
+There are two supported methods for mirroring images onto a mirror registry:
+
+* Using the oc-mirror OpenShift CLI (`oc`) plug-in (recommended)
+
+* Using the oc adm release mirror command
+
+Choose one of the supported options below.
```

Comment:
Do you think it would be acceptable if I were to remove this sentence entirely, and then replace the sentence on line 48 with the one below?

"Choose one of the two supported methods for mirroring images onto a mirror registry:"

Reply:
I think it depends on whether or not this is a procedure or concept. I can see an argument for both options. As is, your heading title suggest procedure. If it is a procedure then I think it is fine as is, but you may want to rework some of the wording above to clarify. If it is a concept, I think the sentence would need to be reworked to introduce the procedures below.

Reply:
@skopacz1 instead of "below", you can use "following".

Reply:
@JoeAldinger I see what you mean. The purpose of listing the two supported options was basically to create some place where I can explicitly mention that users are only supposed to do one of the two options and not both. So I think it's sort of like an intro to a procedure in that sense.

And thanks @bergerhoffer, I was hoping that I could use "following" instead but forgot to ask about it when I made my comment! I'll find some rewording that uses that instead.